testsuite/standalone/src/test/java/org/jboss/as/test/integration/management/cli/BasicOpsTestCase.java

```diff
@@ -72,26 +72,24 @@ public void testLs() throws Exception {
     public void testReadAttribute() throws Exception {
         CLIWrapper cli = new CLIWrapper(true);
         {
-            cli.sendLine("read-attribute namespaces");
-            String output = cli.readOutput();
-            assertFalse(output.contains("Unknown attribute 'namespaces'"));
+            assertTrue("Failed to read attribute: " + cli.readOutput(),
+                    cli.sendLine("read-attribute namespaces", false));
```

Comment:
The way I see the test, it seems that true should be passed to sendLine method call in order to ignore error and have the return value be true if no error, false is error occurred.